masterlist.yaml

```diff
@@ -345,17 +355,6 @@ globals:
     subs: [ 'Place Everywhere' ]
     # url: [ 'https://www.nexusmods.com/fallout4/mods/9424' ]
     condition: 'file("F4SE/Plugins/place.dll") and not file("../f4se_loader.exe")'
-  - type: error
-    content:
-      - lang: en
-        text: 'Your installed version of Place Everywhere is not compatible with your version of Fallout 4.'
-      - lang: da
-        text: 'Din installerede version af Place Everywhere er ikke kompatibel med din version af Fallout 4.'
-      - lang: ja
-        text: 'インストールされているPlace Everywhereは使用中のバージョンのFallout 4との互換性がありません。'
-      - lang: de
-        text: 'Ihre installierte Version von Place Everywhere ist nicht kompatibel mit Ihrer Version von Fallout 4.'
-    condition: 'file("F4SE/Plugins/place.dll") and ((version("F4SE/Plugins/place.dll", "1.18.0.1111", >=) and not version("../Fallout4.exe", "1.10.111.0", ==)) or (version("F4SE/Plugins/place.dll", "1.18.0.1111", <) and version("../Fallout4.exe", "1.10.111.0", ==)))'
```

Comment:
Since these two messages where almost identical. 
I felt it made sense to just combine them into one with subs.

Reply:
Your reasoning for why you're changing something should ideally go into the commit message. You can always tell *what* you've changed by looking at the diff of the commit, but you can't tell *why*. (That doesn't mean you shouldn't also note it in the PR, but if you feel like it needs further clarification in the PR, it likely also does in the commit itself. :))

Reply:
Okay, It would be more useful to have it in the commit message.
I'll add any thoughts or reasons for changes to the commit messages in future.